.gitignore

```diff
@@ -0,0 +1,41 @@
+#### Maven
```

Comment:
Use `/target` etc for items that should only be ignored if they are located in the root of the project. It is safer to be conservative in what you ignore.